app/assets/stylesheets/components/_email-signup-section.scss

```diff
@@ -0,0 +1,26 @@
+.app-c-email-signup--with-background-and-border {
+  padding: govuk-spacing(6);
+  background-color: govuk-colour("light-grey", $legacy: "grey-4");
+  border: 1px solid $govuk-border-colour;
+}
+
+.app-c-email-signup--no-intro .govuk-link {
```

Comment:
This class is called `no-intro` but the component guide refers to it as `Link only`. Can we name them something similar for clarity? Would `link-only` work?